invenio_rdm_records/records/jsonschemas/records/record-v1.0.0.json

```diff
@@ -688,25 +672,10 @@
       }
     },
 
-    "system": {
-      "type": "object",
-      "description": "System information.",
-      "properties": {
-        "notes": {
-          "type": "array",
-          "items":{
-              "type": "object",
-              "properties": {
-                  "created_by": {"$ref": "#/definitions/agent-user"},
-                  "note": {"type": "string"},
-                  "timestamp": {
-                    "description": "ISO8601 formatted date time stamp.",
-                    "type": "string",
-                    "format": "date-time"
-                  }
-              }
-          }
-        }
+    "notes": {
```

Comment:
Shelved: We have to think a bit more. I don't like have this as top-level field. Seems wrong to me....

Reply:
I agree, but `system` wasn't really accurate.